src/main/java/Department.java

```diff
@@ -0,0 +1,20 @@
+import java.util.ArrayList;
+
+
+public class Department {
```

Comment:
Can a Department organize events ? If it can, then there should be a relation between Dept and Event